moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -797,31 +809,53 @@ bool IKFastKinematicsPlugin::searchPositionIK(const geometry_msgs::Pose& ik_pose
   {
     ROS_DEBUG_STREAM_NAMED("ikfast", "No need to search since no free params/redundant joints");
 
-    // Find first IK solution, within joint limits
-    if (!getPositionIK(ik_pose, ik_seed_state, solution, error_code))
+    std::vector<geometry_msgs::Pose> ik_poses;
+    ik_poses.push_back(ik_pose);
```

Comment:
nitpick: `std::vector<geometry_msgs::Pose> ik_poses(1, ik_pose)` is shorter and could (in theory) optimize better.